siddhant/random-wallpaper-wallhaven.py

```diff
@@ -0,0 +1,22 @@
+import requests, bs4, os, random
+url = 'https://alpha.wallhaven.cc/toplist'
+dir_name = 'wallpapers-wallhaven-toplist'
+if not os.path.exists(dir_name):
+    os.makedirs(dir_name)
+print('Downloading Page %s......' % url)
+res = requests.get(url, verify=False)
+res.raise_for_status()
+soup = bs4.BeautifulSoup(res.text, 'html.parser')
+top_images_list = soup.select('section ul li figure a[class="preview"]')
+if top_images_list == []:
+    print('Could not find images!')
+else:
+    random_image_url = 'https:' + '//wallpapers.wallhaven.cc/wallpapers/full/wallhaven-' + top_images_list[random.randint(1, len(top_images_list))].get('href')[38:] + '.jpg'
```

Comment:
From where did you get the number `38`? Also, why is it hardcoded? Does it remain same _each and everytime_?

Reply:
Yes, remains same every time.

Reply:
"https://alpha.wallhaven.cc/wallpaper/661501" 37 characters before 661501. Same for all the wallpapers.

Reply:
Okay, let's merge this in for now.